Rubberduck.VBEEditor/SafeComWrappers/SafeComWrapper.cs

```diff
@@ -54,8 +54,17 @@ public void Release(bool final = false)
                 else
                 {
                     _rcwReferenceCount = Marshal.ReleaseComObject(Target);
-                    _logger.Trace($"Released COM wrapper of type {this.GetType()} with remaining reference count {_rcwReferenceCount}.");
+                    if (_rcwReferenceCount >= 0)
+                    {
+                        _logger.Trace($"Released COM wrapper of type {this.GetType()} with remaining reference count {_rcwReferenceCount}.");
+                    }
+                    else
+                    {
+                        _logger.Warn($"Released COM wrapper of type {this.GetType()} whose underlying RCW has already been released from outside the SafeComWrapper.");
```

Comment:
I think the warning message should still log the new `_rcwReferenceCount` to aid in tracking.